highlights/gf.md

```diff
@@ -0,0 +1,83 @@
+
+Singapore decided to continue with **-45 - Doll of Ai-Ren**. The one and only jack map of this rice pool, 143BPM continuous jacking with the ending being the dense chordjacks, as well as the 4 consecutive jacks in 1 column in some certain parts, the lead gradually shifted towards Singapore while both teams are still keeping the high accuracy. The last half began with the trade misses of riunosk and Molli, the hands were introduced more in the map, then Molli started to not be able to endure their accuracy on the map, dropped to 98% while all three of Singaporeans were still maintaining 99%, resulted in another point Singapore took from their own pick. The score is now 4-3.
+
+China then responded the continuation with their protect: **Shiraishi - Makoto. Sennen Joou**, the Hybrid map of all time mapped to the legendary song which were well known to the osu!mania community. In this approach, they were represented with quads and jacks convoluted with staggered releases alternating with single streams and there was the introduction of light handstreams, dense chordjacks as well as LN coordiation in the parts containing 1/3 snaps. It was started with a chaotic mess of misses due to the unusual pattern of the map, but out of this chaos, it was resulted in China took a lead. TheOPMeme's accuracy dropped down to 97% before the handstream part before the recovery to 98% again after it. Then the 1/3 snaps started to kick in, there were a plenty of misses from China team, another miss from Zyuuu took the lead to Singapore and it was an opportunity for them to steal this pick as they still had 2 players FC'ing the map. Transitioned to the jacks with staggered releases alternated with single streams again, TheOPMeme missed again, led to the slight lead from China, then there came the miss from Zyuuu which led to the full uncertainty of how the point was determined. Neon-Hooray and TheOPMeme however broke at the ending of the song, China were able to secure their pick once again, and the score is now tied at 4-4.
+
+Singapore pulled out the pro gamer move by picking the My Chemical Romance song, **Welcome To The Black Parade**, as the **Lofty's Ending**, the relentless speed stream map testing how much the players can carry on. The lead gradually goes to the Singapore side as they know how to carry on, and it became more explicit that Singapore will carry this point. And certainly, they took it away, with the astounding lead of 95738 points, taking a lead with their own pick again to 5-4.
```

Comment:
'pro gamer' sounds pretty weird, maybe something more relevant like an 'unexpected' pick would make more sense

Reply:
alright, actually i will change to "another move" because it's not really that unexpected